docs/architecture/baking.md

```diff
@@ -31,18 +31,29 @@ To become a baker, you must create an account to act as a _delegate_, which is a
 Similarly, when bakers stop baking, their stake is unlocked after a certain number of cycles.
+While it is locked, the security deposit may be slashed by the protocol if the baker misbehaves (e.g., proposes or attests two different blocks for the same level).
 
-The delegate must stake at least least 6,000 tez, either from its own account or from tez that delegators delegate to it.
-The more tez a delegate has, both from its own account and from delegators, the more likely it is to be selected to bake a block and thus receive the rewards.
+The delegate must stake at least least 600 tez from its own account,
+and must also represent at least 6,000 tez of "active stake", which is computed
+from the tez in its own account and from the tez of all its delegators accounts.
+The more active stake a delegate represents, the more likely it is to be selected to bake a block and thus receive the rewards.
 
 Bakers must run at least one Tezos node and a baker daemon to go with it.
-These daemons must run at all times with a stable internet connection, because inactive bakers are automatically removed from the network.
+These daemons must run at all times with a stable power source and internet connection, because inactive bakers are automatically removed from the network.
 
 ## Delegating to a baker
 
-If you don't have enough tez to become a baker or don't want to run a baking node, you can delegate tez to a baker, which makes you a _delegator_.
-The delegate doesn't have control over your tez and you can spend your tez at any time or withdraw your delegation, but the tez that you delegate counts toward the amount of tez that the delegate has for baking purposes.
-Delegators receive a share of the delegate's rewards in proportion to the amount of tez that they delegate.
+If you don't have enough tez to become a baker or don't want to run a baking node, you can choose a baker as your delegate, which makes you a _delegator_.
+The delegate doesn't have control over your tez and you can spend your tez at any time or withdraw your delegation, but half of the tez that you delegate counts toward the amount of active stake that the delegate has for baking purposes.
+Delegators receive a share of the delegate's rewards in proportion to the amount of tez that they currently have.
+
+## Staking for a baker
+
+If you want to get a bigger share of your delegate's rewards, you can temporarily stake any amount of tez from your account on behalf of your baker.
+All the tez you stake contribute to the baker's security deposit, and are locked under the same conditions as the baker's own staked tez, hence are slashed if the baker misbehaves.
+In exchange of agreeing to temporarily lock these tez and sharing the risks with your delegate, you get a bigger share of the delegate's rewards.
+
+Later on, you may choose to unstake part or all of the staked tez, that will be unlocked after a certain delay.
```

Comment:
```suggestion
Later on, you may choose to unstake part or all of the staked tez, that will be unlocked after a certain delay.

To stake tez, an account starts by delegating its tez to a baker's account as usual.
Then it stakes a certain amount of that same tez to the baker.
An account always delegates all of its tez to its delegate, but it can stake any amount of that tez, which allows the account to earn staking rewards on some frozen tez while leaving some delegated tez liquid for immediate use.

## Becoming a delegator or staker

To start delegating and staking, use the app at https://stake.tezos.com to first become a delegator and then a staker.
```

Reply:
I applied this suggestion manually, because some parts were redundant with the previous paragraph.
(this one, too)